pkg/report/report.go

```diff
@@ -83,6 +83,9 @@ func Write(ctx context.Context, rep *Report, opt flag.Options, fromCache bool) e
 		return err
 	}
 
+	// combine results without a target for consistency of the result
+	filtered = combineResults(filtered)
```

Comment:
What's this for? Should we do it as a separate feature PR?

Reply:
Results can have multiple records without a target in different order which sometimes causes tests to fail. This function simply merges the results without a target and sorts the misconfigs by AVDID.